scripts/check

```diff
@@ -8,8 +8,6 @@ export SOURCE_FILES="httpcore tests"
 
 set -x
 
-${PREFIX}isort --check --diff --project=httpcore $SOURCE_FILES
-${PREFIX}black --exclude '/(_sync|sync_tests)/' --check --diff --target-version=py37 $SOURCE_FILES
-${PREFIX}flake8 $SOURCE_FILES
+${PREFIX}ruff check $SOURCE_FILES
```

Comment:
I think the `--show-source` option is nice here

Reply:
Yes, thank you